cmd/katapult/config_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/krystal/katapult-cli/config"
+)
+
+const mockConfigFormat = `---
+api_key: %s
+api_url: %s
+
+`
+
+func TestConfig(t *testing.T) {
+	conf, err := config.New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	tests := []struct {
+		apiKey string
+		apiURL string
+	}{
```

Comment:
Usually I'd expect a test table to include some sort of expected output within the table, so its really clear what the expected behaviour is.

Reply:
Agreed. I think a `name` field with a descriptive text for each test case would help as well. Cause right now it's rather difficult to quickly grasp what this test is actually testing.